src/devtools/client/webconsole/components/Output/ConsoleOutput.js

```diff
@@ -69,12 +72,55 @@ class ConsoleOutput extends Component {
     };
   }
 
+  constructor(props) {
+    super(props);
+    this.maybeScrollToBottom = this.maybeScrollToBottom.bind(this);
+  }
```

Comment:
we can drop `this.maybeScrollToBottom.bind` in favor of 

```js
maybeScrollToBottom = () => {
}
```

https://babeljs.io/docs/en/babel-plugin-proposal-class-properties